README.md

```diff
@@ -16,6 +16,23 @@ of vulnerability reports.
 This project is developed by [Trail of Bits](https://www.trailofbits.com/) with
 support from Google. This is not an official Google product.
 
+## Index
+
+* [Features](#features)
+* [Installation](#installation)
+  * [Third-party packages](#third-party-packages)
+  * [GitHub Actions](#github-actions)
+  * [`pre-commit` support](#pre-commit-support)
+* [Usage](#usage)
+  * [Exit codes](#exit-codes)
+  * [Dry runs](#dry-runs)
+* [Examples](#examples)
+* [Troubleshooting](#troubleshooting)
+* [Security model](#security-model)
+* [Licensing](#licensing)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
```

Comment:
Just an FYI that these won't work when the README is displayed on PyPI: https://github.com/pypa/readme_renderer/issues/169

Reply:
Yeah, that was in the back of my mind...

IMO that's an acceptable tradeoff for now, since we also direct-link to the repo in a few places (like the GitHub Action).